.github/workflows/ci.yml

```diff
@@ -123,4 +123,4 @@ jobs:
         uses: reviewdog/action-golangci-lint@v1
         # uses: docker://reviewdog/action-golangci-lint:v1 # pre-build docker image
         with:
-          github_token: ${{ secrets.github_token }}
\ No newline at end of file
+          github_token: ${{ secrets.github_token }}
```

Comment:
Did the commit in GitHub, it added a newline to the file on its own I guess. Whatever.